combirepo/temporaries.py

```diff
@@ -169,9 +169,9 @@ def mount_firmware(firmware_path):
         image = images[0]
         mount_image(root, image)
     # For 3-parts and 6-parts images:
-    elif len(images) == 3 or len(images) == 6:
+    elif len(images) in [3, 4, 6]:
         __mount_images_triplet(images, root)
```

Comment:
1. `__mount_images_triplet` method doesn't have information about where to mount swap part, so the resulting image will not have swap part. Is it valid for your use case?
2. Is the name of swap part one of "ramdisk.img", "ramdisk-recovery.img", "modules.img"? If not, we'll have exception at line 140.

Reply:
1. Yes, it is ok.
2. Yes, it is modules.img

Reply:
OK, then it LGTM.